SubmittedAssignments/Raveesh_Rajput/preview.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Visiting Card</title>
+    <link rel="stylesheet" href="style.css">
+
+</head>
+<body>
+        <div class="card-wrapper card-front">
+             <!-- Card Background -->
+            <div class="card1-background">
+                <!-- Customize background elements as needed -->
+            </div>
+            <!-- Content of Visiting Card -->
+            <div class="card1-content">
+                <div class="party-name">Ravish Rajput</div>
+                <div class="line"></div>
+                <div class="party-job-title">
+                    <p>Full Stack Developer</p>
+                </div>
+                <div class="company">MMEAC</div>
+                 <!-- Additional Contact Information -->
+                <div class="company-logo">
+                    <img class="orgLogo" src="./img/org-logo.png" alt="company-logo">
+                </div>
+                <div class="qr-code">
+                    <img src="./img/qr.png" alt="image-qr">
+                </div>
+                <div class="contact-info">
+                    <div class="phone-icon">
+                        <img class="phone-icon1" src="./icons/phone.png" alt="locationIcon">
+                    </div>
+                    <div class="phone">
```

Comment:
change the class name  from **phone** to **party-phone**

Reply:
changes updated.